common-ui/src/main/res/drawable/ic_platform_windows_16.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
+  <path
+      android:pathData="M7.5,1.5H1.5V7.5H7.5V1.5Z"
+      android:fillColor="#000000"
+      android:fillAlpha="0.84"/>
+  <path
+      android:pathData="M7.5,8.5H1.5V14.5H7.5V8.5Z"
+      android:fillColor="#000000"
+      android:fillAlpha="0.84"/>
+  <path
+      android:pathData="M8.5,1.5H14.5V7.5H8.5V1.5Z"
+      android:fillColor="#000000"
+      android:fillAlpha="0.84"/>
+  <path
+      android:pathData="M14.5,8.5H8.5V14.5H14.5V8.5Z"
+      android:fillColor="#000000"
+      android:fillAlpha="0.84"/>
+</vector>
```

Comment:
Same as before, you need to replace `fillColor` attribute with `?attr/daxColorPrimaryIcon`  and remove alpha in all `<paths/>`

Reply:
thanks for the check @nalcalag 👍